### PR TITLE
sentry integration for tessellate server

### PR DESCRIPTION
**Pull request by nerudadhich**

### CODE REVIEW

File: server/grpc.go [hunk 1]
```diff
@@ -37,6 +39,9 @@ var validator = totp.Validate
 func Grpc() *grpc.Server {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
+	raven.SetDSN(*sentryDsn)
```
Comment (mohandutt134) [inline server/grpc.go]: What if `SENTRY_DSN` is not passed in the configuration, will this code break?

File: server/grpc.go [hunk 2]
```diff
@@ -37,6 +39,9 @@ var validator = totp.Validate
...
+	raven.SetEnvironment(*environment)
```
Comment (mohandutt134) [inline server/grpc.go]: Same, what happens if not passed?

File: server/middleware/sentry.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	"context"
+	"errors"
+	"fmt"
+	"runtime/debug"
+
+	"github.com/getsentry/raven-go"
+	"google.golang.org/grpc"
+)
+
+func SentryInterceptor() grpc.UnaryServerInterceptor {
```
Comment (mohandutt134) [inline server/middleware/sentry.go]: Please write tests for this.

File: server/grpc.go (hunk 2 above)
Comment (nerudadhich) [reply server/grpc.go]: it will capture the panic but won't be able to send it on sentry.

File: server/grpc.go (hunk 1 above)
Comment (nerudadhich) [reply server/grpc.go]: it will capture the panic but won't be able to send it on sentry.

**Pull request merged**